types/2022-08-01/index.d.ts

```diff
@@ -134,7 +134,7 @@ declare module 'stripe' {
   export class Stripe {
     static Stripe: typeof Stripe;
 
-    constructor(apiKey: string, config: Stripe.StripeConfig);
+    constructor(apiKey: string, config?: Stripe.StripeConfig);
```

Comment:
missing from changelog

Reply:
ohh hmm what's this about?

Reply:
I'm going to manually revert this, we want to discuss this a little more before we proceed.